_docs-sources/guides/support.mdx

```diff
@@ -38,33 +38,35 @@ We’re here to assist when you get stuck. Basic support is included with every
     Contact <address>support@gruntwork.io</address> with any questions or
     concerns you may have.
   </Card>
-  <Card
-    title="Email With Urgency"
-    icon="/img/support/email-urgent-icon.svg"
-    href="mailto:urgent-support@gruntwork.io"
-    tags={["pro", "enterprise"]}
-  >
-    Messages sent to <address>urgent-support@gruntwork.io</address> will receive
-    a response within 24 hours.
-  </Card>
-  <Card
-    title="Chat With Us"
-    icon="/img/support/chat-icon.svg"
-    href="#chat-with-us-in-slack"
-    tags={["pro", "enterprise"]}
-  >
-    Converse with Gruntwork engineers in your own <em>private</em> Slack
-    channel.
-  </Card>
-  <Card
-    title="Schedule a Call"
-    icon="/img/support/call-icon.svg"
-    href="http://calendly.com/gruntwork-live-support/one-hour"
-    tags={["enterprise"]}
-  >
-    Speak with a Gruntwork engineer via phone or video chat to get real-time
-    answers to your questions.
-  </Card>
+  {
+    //  <Card
```

Comment:
Reminder to throw in a quick comment here to explain that we _do_ want this code but are waiting for final adoption of the patterns before deploying it.